rm-services.yml

```diff
@@ -263,6 +263,18 @@ services:
       timeout: 10s
       retries: 3
 
+  comms-template:
+    container_name: comms-template
+    image: sdcplatform/comms-template-svc
+    ports:
+     - "${COMMS_TEMPLATE_PORT}:${COMMS_TEMPLATE_PORT}"
+    external_links:
+      - postgres
+    environment:
+      - SQLALCHEMY_DATABASE_URI=postgres://${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/postgres
+      - SECURITY_USER_NAME=${SECURITY_USER_NAME}
+      - SECURITY_USER_PASSWORD=${SECURITY_USER_PASSWORD}
+
```

Comment:
Can you add a healthcheck to this. See https://github.com/ONSdigital/ras-rm-docker-dev/pull/30/files#diff-17dcc0d33fe44edc4296df7c7d228432R260